### ISSUE TITLE
Dependency on AutoCloseable prevents using under Java6

### ISSUE THREAD

**Issue by mikesamuel** [OWASP/java-html-sanitizer#58]
arjunvj3 reports

I see the following exception when using v1.1 and the latest v20151202.2 of this project. The same code used to work with the earlier version (r239). It appears like the `AutoCloseable` interface was introduced in Java 7 and isn't available in Java 6. Is there something that could be done to get this project to work with Java 6?

```
java.lang.NoClassDefFoundError: java/lang/AutoCloseable
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClassCond(ClassLoader.java:631)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:615)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:141)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:283)
        at java.net.URLClassLoader.access$000(URLClassLoader.java:58)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:197)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
        at org.mortbay.jetty.webapp.WebAppClassLoader.loadClass(WebAppClassLoader.java:366)
        at org.mortbay.jetty.webapp.WebAppClassLoader.loadClass(WebAppClassLoader.java:337)
        at org.owasp.html.PolicyFactory.sanitize(PolicyFactory.java:123)
        at org.owasp.html.PolicyFactory.sanitize(PolicyFactory.java:101)
```

Following is the snippet of the code I'm using.

``` java
PolicyFactory _policy =
    Sanitizers.BLOCKS
      .and(Sanitizers.FORMATTING)
      .and(Sanitizers.IMAGES)
      .and(Sanitizers.LINKS)
      .and(Sanitizers.TABLES)
      .and(Sanitizers.STYLES);
_policy.sanitize(toSanitize);
```

**Comment by mikesamuel** [comment]
https://travis-ci.org/OWASP/java-html-sanitizer/jobs/106519461 repeats this.  Will fix.

**Comment by mikesamuel** [comment]
Fixed in [`<version>20160203.1</version>`](http://search.maven.org/#artifactdetails%7Ccom.googlecode.owasp-java-html-sanitizer%7Cowasp-java-html-sanitizer%7C20160203.1%7Cjar)

**Comment by volkertb** [comment]
As I just posted in issue #50, this issue has apparently not been solved in that version after all, at least not under all possible circumstances. I still get basically the same stacktrace as the one above.
